src/client/pages/create.js

```diff
@@ -136,6 +138,21 @@ class Create extends React.Component {
     })
   }
 
+  async copyToClipboard(text) {
+    //Not sure I fully understand binding, should this function be bound to this?
+    return await navigator.clipboard.writeText(text);
+  }
+
+  copyAllLinks() {
+    this.copyToClipboard(
+      "You have been invited to a game of Elevation of Privilege:\n" +
+      Array(this.state.players).fill(0).map((v, i) => {
+        return `${this.state.names[i]}:\t${window.location.origin}/${this.state.gameID}/${i}/${this.state.secret[i]}\n`;
+      }).join("")
```

Comment:
Would it make it more readable to remove the `\n` from the previous line and join via `.join("\n")` ?

Reply:
maybe even join with 2 newlines? Then it's easier for the players to copy their link from the chat?

Reply:
Good idea